### ISSUE TITLE
new ImageComparison constructor is unable find image 

### ISSUE THREAD

**Issue by mrgoroua** [romankh3/image-comparison#180]
following  code:

 ```
String expectedReceiptPath = ".\\ExpectedCashNoTipsReceipt.png";
String actualReceiptPath = ".\\ExpectedCashNoTipsReceipt.png";

ImageComparison imageComparison = new  ImageComparison(expectedReceiptPath, actualReceiptPath);
```

fails with an error:  

```
com.github.romankh3.image.comparison.exception.ImageNotFoundException: 

Image with path = .\ExpectedCashNoTipsReceipt.png not found

```

while file really exists 

see test example

![image](https://user-images.githubusercontent.com/14112575/78057004-b0fad400-738e-11ea-9c2b-0e9215774782.png)

**Comment by romankh3** [comment]
I’ll see it, thank you for your catch.

**Comment by Zyronath** [comment]
Is there a way to work aroung this? I keep having this issue. 
![bug](https://user-images.githubusercontent.com/14099228/79308645-42268a80-7ef9-11ea-8fff-c4ea20e4ee18.png)

**Comment by romankh3** [comment]
Hello, @Zyronath. 
There are a few ways to handle it:
1. I found that this bug reproduces in the case when pictures are in the root dir of the project. If you put them into resources, it must work. I used the same way for testing an app and my images in `src/test/resources` dir.
2. Use another constructor with `BufferedImage` objects instead. 
3. Use '3.2.0' version instead.
4. Contribute to the project and provide the fix for it :) Unfortunately, I don't have enough time to do it now.
